study.md

```diff
@@ -0,0 +1,43 @@
+# mongodb Study
+
+Use your favorite search engine and the provided readings to research and
+respond to the following questions.
+
+In your responses, be sure to cite any relevant sources you consulted in your
+search. We ask you to write responses in your own words in order to see how you
+process what you've read. Please do not respond with direct quotes from source
+material. Instead, digest what you've read and repeat it in your own voice.
+
+## Required Readings
+
+- [NoSQL Databases Explained](https://www.mongodb.com/nosql-explained)
```

Comment:
Good catch! I was about to call you out for it until I realized that's where these belong, according to the `study-template`